source/features/view-last-pr-deployment.tsx

```diff
@@ -0,0 +1,27 @@
+import select from 'select-dom';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+import observeElement from '../helpers/simplified-element-observer';
+
+function init(): void | false {
+	const lastDeployment = select.last('.js-timeline-item [data-url$="deployed"] .TimelineItem-body .btn[target="_blank"]');
+	if (!lastDeployment) {
+		return false;
+	}
+
+	const button = lastDeployment.cloneNode(true);
+	button.classList.add('mr-1');
+	button.textContent = 'View last deployment';
+
+	select('.gh-header-actions')!.prepend(button);
+}
+
+void features.add(__filebasename, {
+	include: [
+		pageDetect.isPRConversation
+	],
```

Comment:
Given that most PRs don't have this, let's not add an observer for them
```suggestion
	],
	exclude: [
		() => !select.exists(deploymentsSelector)
	],
```

Reply:
What if a deployment is triggered for the first time after the person visits the page? Don't worry about it?

Reply:
Yeah. In that slim chance, the link will likely be at the bottom of the page, easily accessible anyway.